api/src/main/java/org/pmiops/workbench/workspaces/resources/UserRecentResourceServiceImpl.java

```diff
@@ -72,24 +83,20 @@ public void updateConceptSetEntry(long workspaceId, long userId, long conceptSet
 
   @Override
   public void updateDataSetEntry(long workspaceId, long userId, long dataSetId) {
-    if (workbenchConfigProvider.get().featureFlags.enableDSCREntryInRecentModified) {
```

Comment:
Now I'm remembering other conversations ... I believe it was also OK to simply add these entries in all envs once the logic made it safe to do so.

Or we could revert this and save these changes for the next PR